content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -186,57 +174,39 @@ To help validation list publishers and other participants in the XRP Ledger netw
 
 - Use your validator key to claim ownership of a domain.
 
-Creating this link establishes strong evidence that you own the validator key and the domain.
+Creating this link establishes strong evidence that you own both the validator key and the domain.
 
 To provide domain verification:
 
-1. Choose a domain name you own that you want to be publicly associated with your validator. You must run a public-facing HTTPS server on port 443 of that domain and you must have access to the private key file associated with that server's TLS certificate. (Note: TLS was formerly called SSL.)
+1. Choose a domain name you own that you want to be publicly associated with your validator. You must run a public-facing HTTPS server on port 443 of that domain and you must have access to the private key file associated with that server's TLS certificate. (Note: [TLS was formerly called SSL](https://en.wikipedia.org/wiki/Transport_Layer_Security).)
 
-2. Share your validator's public key with the public, especially other `rippled` operators. For example, you can share your validator's public key on your website, on social media, in the [XRPChat forum](https://www.xrpchat.com/), or in a press release.
+2. Share your validator's public key with the public, especially other `rippled` operators. For example, you can share your validator's public key on your website, on social media, in the [XRPChat community forum](https://www.xrpchat.com/), or in a press release.
 
 3. Submit a request to have your validator listed in XRP Charts' [Validator Registry](https://xrpcharts.ripple.com/#/validators) using this [Google Form](https://docs.google.com/forms/d/e/1FAIpQLScszfq7rRLAfArSZtvitCyl-VFA9cNcdnXLFjURsdCQ3gHW7w/viewform). Having your validator listed in this registry provides another form of public verification that your validator and domain are owned by you. To complete the form, you'll need the following information:
 
       1. Find your validator public key by running the following on the validator server.
 
-        ```
-        $ /opt/ripple/bin/rippled server_info -q | grep pubkey_validator
-        ```
+            $ /opt/ripple/bin/rippled server_info -q | grep pubkey_validator
```

Comment:
Optional nit: I think the `-q` here is [unnecessary](https://developers.ripple.com/commandline-usage.html#unit-tests) and probably ignored by the server anyway.

Reply:
Sounds great - thank you for the link to more info on this.